.github/workflows/test.yml

```
@@ -0,0 +1,44 @@
+name: fontsan CI
+
+on: push
+
+jobs:
+  build_and_test:
+    name: Build and Test
+    strategy:
+      matrix:
+        os:
+          - "ubuntu-latest"
+          - "macos-latest"
+          - "windows-latest"
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+
+      - name: Cargo build
+        run: cargo build -vv
+
+      - name: Cargo test
+        run: cargo test -vv
+        env:
+          RUST_BACKTRACE: 1
+
+  build_result:
+    name: homu build finished
```

This repository no longer uses homu.

```suggestion
    name: Result
```